src/object.rs

```diff
@@ -44,6 +45,13 @@ extern "C" {
     key: *const Name,
     value: *const Value,
   ) -> MaybeBool;
+  fn v8__Object__DefineOwnProperty(
+    object: &Object,
+    context: *const Context,
```

Comment:
I would swear I commented here, but my comment seems to be nowhere...

It's not necessary to unwrap Local<T> to a raw pointer before passing them as a function argument from Rust to C++.
If possible, try to be consistent between the `extern "C"` declarations on the C++ vs Rust side, which is lacking here:

`fn v8__Object__DefineOwnProperty(object: &Object, context: *const Context, ...) -> MaybeBool;`
`MaybeBool v8__Object__DefineOwnProperty(v8::Object& self, v8::Local<v8::Context> context, ...);`

I know it's probably copypasta but let's start somewhere. 
(cc @ry, @bartlomieju).

Reply:
Yep, I copied it from create_data_property(). I'll see if I have time today to do a (general) clean up.